easybuild/easyconfigs/c/CPMD/CPMD-4.1r3816-intel-2015a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'CPMD'
+version = '4.1'
+
+homepage = 'http://cpmd.org'
+description = """The CPMD code is a parallelized plane wave / pseudopotential
+implementation of Density Functional Theory, particularly designed for
+ab-initio molecular dynamics."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://cpmd.org/download']
```

Comment:
requires registration, so drop `source_urls` and include a comment with a pointer to the download location (and mention that it requires registration)